app/gui2/vite.config.ts

```diff
@@ -16,6 +16,7 @@ const projectManagerUrl = 'ws://127.0.0.1:30535'
 
 const IS_CLOUD_BUILD = process.env.CLOUD_BUILD === 'true'
 const POLYGLOT_YDOC_SERVER = process.env.POLYGLOT_YDOC_SERVER
+const YDOC_DEBUG = process.env.YDOC_DEBUG
```

Comment:
I think this env variable is worth adding to `.env`, don't you think? It'd be easier to review the variables and avoid duplications/reduce the onboarding time

Reply:
You mean https://github.com/enso-org/enso/blob/develop/app/.example.env ? Idk, env variables there seem very much different from what I'm proposing. Similarly `POLYGLOT_YDOC_SERVER` is not there either.

Reply:
We should add both vars to that file IMO, for the reasons Sergei listed.

Reply:
Added